iceoryx2-ffi/ffi/BUILD.bazel

```diff
@@ -20,9 +20,33 @@ filegroup(
     srcs = glob(["**"]),
 )
 
-filegroup(
+
+load("@rules_rust//rust:defs.bzl", "rust_binary")
+
+# load("@cbindgen_source//:BUILD.bazel", "srcs")
+rust_binary(
     name = "cbindgen-cli",
-    srcs = ["@cbindgen//file"],
+    srcs = [
+        "@cbindgen_source//:srcs",
+    ],
+    data = [
+        "@cbindgen_source//:Cargo.toml",
+        "@cbindgen_source//:Cargo.lock",
```

Comment:
Nice. I have to test if this also works with bazel 6.2.

Do you have experience with bazel?

Reply:
I didn't have any experience before, but gained it a lot yesterday and today. The gpt is quite bad and there is no good idea from it at all, I have had to check the docs and stackoverflow.